src/kpro_connection.erl

```diff
@@ -0,0 +1,598 @@
+%%%
+%%%   Copyright (c) 2014-2018, Klarna AB
```

Comment:
Is this largely new code or a copy of old code?  In the former case, make the copyright 2018 only.

Reply:
a copy with a small change-set